<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1021</width>
    <height>681</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="SrcFld">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>폴더 찾기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CreateFld">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>폴더 생성</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DelFld">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>폴더 삭제</string>
   </property>
  </widget>
  <widget class="QLabel" name="imglabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>600</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="imgbtn">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>이미지</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
